cmd/docs.go

```diff
@@ -105,14 +113,25 @@ func resolveStringArray(customField interface{}) []string {
 	return retval
 }
 
-func generateDocs(engine *opa_engine.Enginer) map[string][]PolicyDoc {
+func translator(scmType scm_type.ScmType, module string) string {
+	val, ok := translationMapping[scmType]
+	if ok {
+		translatedModule, ok := val[module]
+		if ok {
+			return translatedModule
+		}
+	}
+	return module
+}
```

Comment:
I'd prefer to avoid nesting:
```
	scmMap, ok := translationMapping[scmType]
	if !ok {
	  return module
	}
        translatedModule, ok := scmMap[module]
        if !ok {
	        return module
	}
	return translatedModule
```